src/py/mat3ra/made/tools/build/defect/builders.py

```diff
@@ -191,18 +188,22 @@ def create_material_with_additional_layers_int(self, material: Material, added_t
             A new Material instance with the added layers.
         """
 
-        new_material = material.clone()
-        termination = Termination.from_string(new_material.metadata.get("build").get("termination"))
-        build_config = new_material.metadata.get("build").get("configuration")
-
-        if build_config["type"] != "SlabConfiguration":
+        slab_build_configuration_dict = material.metadata.get("build").get("configuration")
```

Comment:
Let's put a task to define a MaterialMetadata class and replace all `get` statements with it